app/javascript/profilePreviewCards/MinimalProfilePreviewCard.jsx

```diff
@@ -0,0 +1,60 @@
+import { h } from 'preact';
+
+export const MinimalProfilePreviewCard = ({
+  triggerId,
+  contentId,
+  username,
+  name,
+  profileImage,
+  userId,
+}) => (
+  <div class="profile-preview-card relative mb-4 s:mb-0 fw-medium hidden m:inline-block">
+    <button
+      id={triggerId}
+      aria-controls={contentId}
+      class="profile-preview-card__trigger fs-s px-0 crayons-btn crayons-btn--ghost p-0"
+      aria-label={`${name} profile details`}
+    >
+      {name}
+    </button>
+
+    <div
+      id={contentId}
+      class="profile-preview-card__content crayons-dropdown"
+      style="border-top: var(--su-7) solid var(--card-color);"
+      data-testid="profile-preview-card"
```

Comment:
Question (non-blocking): Is it possible to get the profile preview card without a `data-testid` in an accessible manner?

Reply:
I think the best we could do is to `findByRole...` for the trigger button, and somehow dynamically pull out the `aria-controls` attribute ID then do `cy.get('#someId')` 🙁  

We only use the `data-testid` to scope our queries within the preview card though, and the tests tend to all check e.g. focus being transferred to the right element inside when it opens/closes so I feel like we're still testing that the button/dropdown are properly linked